sar/README.md

```diff
@@ -119,20 +119,22 @@ only additional requirements and mappings to fulfill the CARD4L requirements are
+| card4l:orbit_mean_altitude             | ✓ 1.6.5     | ✗           | `OrbitMeanAltitude`                                          | number                                                  | Platform (mean) altitude in meters (m). |
 | card4l:source_processing_parameters    | ✓ 1.6.6     | ✗           | *various, see below*                                         | [Source Processing Object](#source-processing-object)   | Additional relevant processing parameters, e.g., Range- and Azimuth Look Bandwidth and LUT applied. If not available in machine-readable form, can also be specified in `processing:lineage`. |
 | card4l:source_geometry                 | ✓ 1.6.7     | ✗           | `SourceDataGeometry`                                         | string                                                  | **REQUIRED.** One of `slant-range` or `ground-range`. |
-| card4l:incidence_angle_near_range      | ✓ 1.6.7     | ✗           | `IncAngleNearRange`                                          | number                                                  | **REQUIRED.** Convert to degree, if required.                |
-| card4l:incidence_angle_far_range       | ✓ 1.6.7     | ✗           | `IncAngleFarRange`                                           | number                                                  | **REQUIRED.** Convert to degree, if required.                |
+| card4l:incidence_angle_near_range      | ✓ 1.6.7     | ✗           | `IncAngleNearRange`                                          | number                                                  | **REQUIRED.** In degrees. |
+| card4l:incidence_angle_far_range       | ✓ 1.6.7     | ✗           | `IncAngleFarRange`                                           | number                                                  | **REQUIRED.** In degrees. |
 | card4l:noise_equivalent_intensity      | ✓ 1.6.9     | ✗           | `Estimates` in `NoiseEquivalentIntensity`                    | [Statistics Object](#statistics-object)                 | **REQUIRED.** Fill the object with statistics that are available for the source data, e.g. min, max and mean. Convert each to decibel, if required. |
 | card4l:noise_equivalent_intensity_type | ✓ 1.6.9     | ✗           | `NoiseEquivalentIntensity`, attribute `type`                 | string                                                  | **REQUIRED.** One of `beta0`, `sigma0`, or `gamma0`. |
+| card4l:peak_sidelobe_ratio             | ✓ 1.6.9     | ✗           | `PeakSideLobeRatio`                                          | number                                                  | Peak sidelobe ratio (PSLR) in decibel. |
+| card4l:integrated_sidelobe_ratio       | ✓ 1.6.9     | ✗           | `IntegratedSideLobeRatio`                                    | number                                                  | Integrated sidelobe ratio (ISLR) in decibel. |
```

Comment:
These two need to be added to the JSON Schema, too.

Reply:
Let me know if you are unfamiliar with JSON Schema. Happy to help with the JSON Schema changes.

Reply:
Thanks! I'm not familiar but seeing what has been done to similar properties helps 🙂
Hope I didn't miss anything:
0b4ec74

Reply:
Looks good. The only question one may raise is whether the value should be >= 0 (minimumExclusive) or > 0 (minimum).

Reply:
Hmm I'd say > 0 (minimum), but I'm not 100% sure.

Reply:
Actually, I made a mistake here. >0 is minimumExclusive, but if we are not sure we can also keep it as it is. Not a big deal.